docs/src/examples/scikit-learn.rst

```diff
@@ -0,0 +1,112 @@
+********************
+Scikit-learn
+********************
+.. Might also think of moving this file to examples/ then we an example auto contained in the
+   repository. We invoke this file from index.rst
+
+In this example, we're going to demonstrate how Oríon can be integrated to a minimal model using
+scikit-learn [scikit]_ on the iris dataset [iris]_. The files mentioned in this example are
```

Comment:
I think it would be more useful here to provide a URL to scikit instead of a citation. For iris it would be great to provide an url to the dataset as well. Similarly, the line below should point to the examples folder in the repo on github, so the user can directly go see.

Reply:
Good catch! I'm going to change it.